### PR TITLE
Update README.md - DropDown

### PR DESCRIPTION
**Pull request by AmenaNajeeb**
Created Number 22. DropDown

The users will be able to see the detailed message only when they click on the summary.

### CODE REVIEW

**Review by atapas — COMMENTED** [review]
@AmenaNajeeb thanks for putting it. Can you please create an issue for the PR so that we have a track of changes?

**Comment by AmenaNajeeb** [comment]
Yes Sir.

File: README.md [hunk 1]
```diff
@@ -423,4 +424,30 @@ Did you know I have [Website](https://tapasadhikary.com)?
 
 For more information on how to write mathematical expressions, [visit this page](https://en.wikibooks.org/wiki/LaTeX/Mathematics).
 
+## DropDown
+
+**Syntax**
+```
+<details open>
```
Comment (atapas) [inline README.md:431]: @AmenaNajeeb Can we please show the example without `open`?

File: README.md [hunk 2]
```diff
@@ -423,4 +424,30 @@ Did you know I have [Website](https://tapasadhikary.com)?
 
 For more information on how to write mathematical expressions, [visit this page](https://en.wikibooks.org/wiki/LaTeX/Mathematics).
 
+## DropDown
+
```
Comment (atapas) [inline README.md:428]: Here we can add a sentence explaining what `summary` and `details` and talk about the `open` attribute.

**Review by atapas — CHANGES_REQUESTED** [review]
@AmenaNajeeb Thanks for putting this PR. A few comments.

**Review by AmenaNajeeb — COMMENTED** [review]
All suggestions are considered and changes are made.

**Comment by atapas** [comment]
> All suggestions are considered and changes are made.

@AmenaNajeeb I do not see the changes though... Have you pushed?

**Review by AmenaNajeeb — COMMENTED** [review]
Updated File.

Screenshot (2135)
Screenshot (2136)

**Comment by atapas** [comment]
> Updated File.
> 
> Screenshot (2135) Screenshot (2136)

Yes but it is in your forked Repo, you need to push it to UPSTREAM again to get reflected in the PR @AmenaNajeeb

**Comment by AmenaNajeeb** [comment]
Pushed the Updated File on your Original Repo. 

Please check, Sir.

I referred this link:  https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-for-a-fork

**Comment by atapas** [comment]
@AmenaNajeeb well, I still don't see your changes in the upstream repo. You can check it here: https://github.com/atapas/markdown-cheatsheet/pull/12/files

Can you please paste all the commands you are trying to push your changes to upstream?

**Comment by AmenaNajeeb** [comment]
Yes Sir.
The changes that I had done were on my repo in GitHub.
I first opened GitBash and cloned by repo to my local computer.
Then changed the directory path to this cloned repo.
Then..
$git remote add atapas git://github/atapas/markdown-cheatsheet
Then
$git remote -v

**Review by AmenaNajeeb — COMMENTED** [review]
Line 26 and Lines 426 to 485 are added to explain DropDown.

**Review by atapas — APPROVED** [review]

**Comment by AmenaNajeeb** [comment]
Pushed the Updated File on your Original Repo.

Please check.

Referred this link:
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-for-a-fork

**Pull request merged**